### PR TITLE
Add top token stats

### PR DESCRIPTION
**Pull request by mayhem**
The same as the previous PR, but for top tokens. Also add test.sh and this version uses column names and not column indexes.

### CODE REVIEW

File: metabrainz/model/access_log.py [hunk 1]
```diff
@@ -188,3 +188,45 @@ def top_ips(cls, days=7, limit=None):
...
+            list for commercial users and another list for non-commercial users who
+            are not in good standing. good standing for non-commercial users means
+            that they are verified good non-commercial users.
+
+        Args:
+            days: Number of past days to include in the query
+            limit: Max number of items to return.
+
+        Returns:
+            Tuple of (non_commercial, commercial) lists of [ip_address, token, musicbrainz_id, user_id, contact_name, contact_email]
```
Comment (paramsingh) [inline metabrainz/model/access_log.py]: This function doesn't have ip addresses, right?

File: metabrainz/templates/admin/stats/top-tokens.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+       Users in good standing are not listed here.
+    </p>
+    {{ user_list(non_commercial) }}
+
+    <h2>Commercial users</h2>
+    {{ user_list(commercial) }}
+
+{% endblock %}
+
+{% macro user_list(users) -%}
```
Comment (paramsingh) [inline metabrainz/templates/admin/stats/top-tokens.html]: Could we reuse the macro from the other template in the previous PR?

Comment (mayhem) [reply metabrainz/templates/admin/stats/top-tokens.html]: We can't; there is one columns less in this view.

File: metabrainz/model/access_log.py [hunk 3]
```diff
@@ -147,3 +147,86 @@ def top_downloaders(cls, limit=None):
...
             query = query.limit(limit)
         return query.all()
+
+    @classmethod
+    def top_ips(cls, days=7, limit=None):
+        """
+            Generates two list of most active ip addresses in the last days. One
+            list for commercial users and another list for non-commercial users who
+            are not in good standing. good standing for non-commercial users means
+            that they are verified good non-commercial users.
```
Comment (reosarevok) [inline metabrainz/model/access_log.py]: Generates two list**s** of **the** most active **IP** addresses in the last days. One
            list for commercial users and another list for non-commercial users who
            are not in good standing. **G**ood standing for non-commercial users means
            that they are verified good non-commercial users.

File: metabrainz/model/access_log.py [hunk 4]
```diff
@@ -147,3 +147,86 @@ def top_downloaders(cls, limit=None):
...
+        return (non_commercial, commercial)
+
+
+    @classmethod
+    def top_tokens(cls, days=7, limit=None):
+        """
+            Generates two list of most active token in the last days. One
+            list for commercial users and another list for non-commercial users who
+            are not in good standing. good standing for non-commercial users means
+            that they are verified good non-commercial users.
```
Comment (reosarevok) [inline metabrainz/model/access_log.py]: Generates two list**s** of **the** most active token**s** in the last days. One
            list for commercial users and another list for non-commercial users who
            are not in good standing. **G**ood standing for non-commercial users means
            that they are verified good non-commercial users.

**Pull request merged**